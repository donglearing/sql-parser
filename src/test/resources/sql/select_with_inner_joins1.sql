select t.a, f.b from T as t inner join F as f on t.a = f.b and f.b > 100 inner join H as h on f.c = h.c;