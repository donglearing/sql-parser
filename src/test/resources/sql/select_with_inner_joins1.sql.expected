SELECT t.a, f.b FROM T AS t INNER JOIN F AS f ON t.a = f.b AND f.b > 100 INNER JOIN H AS h ON f.c = h.c